plugin/core/workspace.py

```diff
@@ -13,7 +13,7 @@ def get_project_path(window: 'Any') -> 'Optional[str]':
     """
     Returns the first project folder or the parent folder of the active view
     """
-    if len(window.folders()):
+    if len(window.folders()) == 1:
```

Comment:
I downloaded RLS and can reproduce your problem locally. Could you try using this instead?

```
import os
try:
    from typing import List, Optional, Any, Iterable
    assert List and Optional and Any and Iterable
except ImportError:
    pass

from .logging import debug
from .types import ViewLike


def get_filename_from_view(view: ViewLike) -> 'Optional[str]':
    if not view:
        debug("No view is active in current window")
        return None  # https://github.com/tomv564/LSP/issues/219
    filename = view.file_name()
    if not filename:
        debug("Couldn't determine project directory since no folders are open",
              "and the current file isn't saved on the disk.")
    return filename


def get_directory_name(view: ViewLike) -> 'Optional[str]':
    filename = get_filename_from_view(view)
    if filename:
        project_path = os.path.dirname(filename)
        return project_path
    return None


def find_path_among_multi_folders(folders: 'Iterable[str]',
                                  view: ViewLike) -> 'Optional[str]':
    filename = get_filename_from_view(view)
    if not filename:
        return None
    folders = [os.path.realpath(f) for f in folders]
    file = view.file_name()
    if not file:
        return None
    file = os.path.realpath(file)
    while file not in folders:
        file = os.path.dirname(file)
        if os.path.dirname(file) == file:
            # We're at the root of the filesystem.
            file = None
            break
    debug('project path is', file)
    return file


def get_project_path(window: 'Any') -> 'Optional[str]':
    """
    Returns the project folder or the parent folder of the active view
    """
    if not window:
        return None
    num_folders = len(window.folders())
    if num_folders == 0:
        return get_directory_name(window.active_view())
    elif num_folders == 1:
        folder_paths = window.folders()
        return folder_paths[0]
    else:  # num_folders > 1
        return find_path_among_multi_folders(
            window.folders(),
            window.active_view())
```

also we should really get some unit tests going for get_project_path

Reply:
This works perfectly! I'll throw together some tests and update this.

Reply:
There is a MockWindow class sitting in test_windows.py that you can use

Reply:
just run `tox -q` on the command-line to run all tests (except for the unittesting plugin suite)